examples/rerun/pixi.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "rerun"
+version = "0.1.0"
+description = "Example package to run the rerun-sdk"
+authors = ["Ruben Arts <ruben@prefix.dev>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "win-64", "osx-64"]
+
+[tasks]
+start = "pixi run python dna_exmaple.py "
```

Doesnt just 

```suggestion
start = "python dna_exmaple.py"
```

work?

---

(It does on my machine :) )

---

Lolll funny that it still works. Pixi is already in my muscle memory 🤣